server/embedded/src/main/resources/log4j2.xml

```diff
@@ -339,6 +337,6 @@ account managers so that we can coordinate edits with other customized copies of
             <AppenderRef ref="CONSOLE"/>
         </Logger>
 
-        <Logger name="NoOpLogger" level="OFF" additivity="false"/>
+        <Logger name="NoOpLogger" level="OFF" />
```

Comment:
I don't know who added this or why, but guessing that `addivitity="false"` is intentional